pom.xml

```diff
@@ -501,11 +506,10 @@
         <plugins>
             <plugin>
                 <artifactId>maven-site-plugin</artifactId>
-                <version>3.5.1</version>
```

Comment:
Github seems not to render this correct. I updated it to 3.7.1

Reply:
There is an update to 3.7.1 upper, line 179 ...